nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -527,7 +536,7 @@ contract StakingEscrow is Issuer {
     * @param _value Amount of tokens which will be locked
     * @param _periods Amount of periods during which tokens will be locked
     **/
-    function lock(uint256 _value, uint16 _periods) public onlyStaker {
+    function lock(uint256 _value, uint16 _periods) external onlyStaker {
```

Comment:
Why all these changes from `public` to `external`?

Reply:
I prefer when function has the most strict modifier, and those methods logically only for calling from outside. No reason from optimization perspective